samples/cellular/modem_shell/README.rst

```diff
@@ -819,6 +819,26 @@ You can use the ``heap`` command to print kernel and system heap usage statistic
      free:              160
      allocated:          88
 
+----
+
+GPIO pin pulse counter
+======================
+
+You can use the ``gpio_count`` command to count pulses on a given GPIO pin.
```

Comment:
We should probably add to the beginning and the first sentence modified to fit better:

MoSh command: ``gpio_count``

Reply:
I did not notice that most of the commands are documented like this. Not all are though, at least `sleep`, `uart` or `heap`. I will update all of these.